src/actions/workspace.ts

```
@@ -501,4 +501,88 @@ export const deleteWorkspace = async (workspaceId: string) => {
+        })
+
+        return {
+          status: 200,
+          data: {
+            message: 'Folder and all its videos deleted successfully',
+            workspaceId: folder.workSpaceId
+          }
+        }
+      } catch (error) {
+        // Transaction will automatically rollback
+        throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)